Gamma_Analysis.py

```diff
@@ -0,0 +1,341 @@
+    alongside its uncertainty.
+    """
+    emission_rate = [net_area[0]/(efficiency*livetime),
+                     net_area[1]/(efficiency*livetime)]
+    return emission_rate
+
+
+def Isotope_Activity(Isotope, emission_rates, emission_uncertainty):
+    """
+    Isotope_Activity will determine the activity of a given radioactive isotope
+    based on the emission rates given and the isotope properties. It takes an
+    Isotope object and a given set of emission rates and outputs an activity
+    estimate alongside its uncertainty.
+    """
+    Branching_ratio = Isotope.list_sig_g_b_r
+    Activity = []
+    Uncertainty = []
+    for i in range(len(Branching_ratio)):
+        Activity.append(emission_rates[i]/Branching_ratio[i])
+        Uncertainty.append(emission_uncertainty[i]/Branching_ratio[i])
+    Isotope_Activity = np.mean(Activity)
+    Activity_uncertainty = np.mean(Uncertainty)
+    results = [Isotope_Activity, Activity_uncertainty]
+    return results
+
+
+def peak_measurement(M, energy):
+    """
+    Takes in a measured spectra alongside a specific energy and returns the net
+    area and uncertainty for that energy.
+    """
+    energy_axis = M[0]
+    M_counts = M[1]
+
+    # Rough estimate of FWHM.
+    FWHM = 0.05*energy**0.5
+
+    # Peak Gross Area
+
+    start_peak = 0
+    while energy_axis[start_peak] < (energy - FWHM):
+        start_peak += 1
+
+    end_peak = start_peak
+    while energy_axis[end_peak] < (energy + FWHM):
+        end_peak += 1
+
+    gross_counts_peak = sum(M_counts[start_peak:end_peak])
+
+    # Left Gross Area
+
+    left_peak = energy - 2*FWHM
+
+    left_start = 0
+    while energy_axis[left_start] < (left_peak - FWHM):
+        left_start += 1
+
+    left_end = left_start
+    while energy_axis[left_end] < (left_peak + FWHM):
+        left_end += 1
+
+    gross_counts_left = sum(M_counts[left_start:left_end])
+
+    # Right Gross Area
+
+    right_peak = energy + 2*FWHM
+
+    right_start = 0
+    while energy_axis[right_start] < (right_peak - FWHM):
+        right_start += 1
+
+    right_end = right_start
+    while energy_axis[right_end] < (right_peak + FWHM):
+        right_end += 1
+
+    gross_counts_right = sum(M_counts[right_start:right_end])
+
+    # Net Area
+
+    net_area = gross_counts_peak - (gross_counts_left + gross_counts_right)/2
+
+    # Uncertainty
+
+    uncertainty = abs((gross_counts_peak +
+                      (gross_counts_left + gross_counts_right) / 4)) ** 0.5
+
+    # Returning results
+
+    results = [net_area, uncertainty]
+
+    return results
+
+
+def Background_Subtract(M, B):
+    """
+    Background_Subtract will subtract the background spectrum
+    from a given measurement. This will return the energy bins using a given
+    energy calibration as well as the background subtracted counts of a
+    measurement. Since both spectra come from the same detector, this assumes
+    the energy calibrations are the same.
```

Comment:
Please clarify in the docstring what `M` and `B` are. The user shouldn't have to guess which is background, which is measurement, and whether they are spectrum objects or NumPy arrays or what.